SocialCareCaseViewerApi.Tests/V1/Domain/WarningNoteTests.cs

```diff
@@ -16,87 +16,42 @@ public void SetUp()
         }
```

Comment:
I think we can delete this class.

We have been slowly deleting classes where the test is just testing the default property values.

Reply:
👍🏾

Reply:
Will do this in a separate PR.